resources/python-examples.md

```diff
@@ -0,0 +1,30 @@
+<p align="center">
+
+## Contents
+- [Documentation](#documentation-links)
```

Comment:
Please make the names in the Contents list exactly the same as the headers further down (so 'documentation links' instead of only 'documentation')